### PR TITLE
Feature/card

### PR DESCRIPTION
**Pull request by jean619215**

### CODE REVIEW

File: src/components/Card/Card.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+import { BasicDivStyled, BasicProps } from '../Basic/Basic'
+
+export interface CardStyledProps extends BasicProps {
+  minWidth?: string
+  minHeight?: string
+}
+
+const CardStyled = styled(BasicDivStyled)<CardStyledProps>`
+  background: ${({ theme }) => theme?.colors?.backgroundColor ?? '#F6F6F6'};
+  min-width: ${({ minWidth }) => minWidth ?? '300px'};
```
Comment (crispyfisherman) [inline src/components/Card/Card.tsx]: Does the default of minWidth and minHeight suitable for mobile screens?

Comment (jean619215) [reply src/components/Card/Card.tsx]: Actually not, I think the default setting should be improve
give me a second

Comment (jean619215) [reply src/components/Card/Card.tsx]: BTW, @danhsucowboy  how about your idea?
the default setting for mobile or screens or both?

Comment (crispyfisherman) [reply src/components/Card/Card.tsx]: For both!
We can write a way like this for mobile:
``` Typescript
const CardStyled = styled(BasicDivStyled)<CardStyledProps>`
 .
 .
 .
  @media only screen and (max-width: 960px) {
    min-width: ${({ minWidth }) => minWidth ?? '....px'};
    min-height: ${({ minHeight }) => minHeight ?? '....px'};
  }
`
```
If the size can't be sure is fine, we can set the correct number after the confirmed design shows up.

Comment (jean619215) [reply src/components/Card/Card.tsx]: but last time we discuss about this
`@media only screen and (max-width: 960px)` it should't write in UI-layer

Comment (jean619215) [reply src/components/Card/Card.tsx]: Maybe we can set the size , like `S, M ,L ` and let user use different size in each screen size

**Review by sc0Vu — APPROVED** [review]
LGTM

**Pull request merged**